projects/adxrs290-pmdz/src/platform/aducm3029/parameters.h

```diff
@@ -60,7 +61,7 @@
 #define UART_BAUDRATE	115200
 
 #define SPI_DEVICE_ID   1
-#define SPI_BAUDRATE    1000000
+#define SPI_BAUDRATE    4000000
```

Comment:
Is this done on purpose?

Reply:
yes, increasing the SPI baud-rate will allow the whole interrupt-based system to work as expected

Reply:
ok...